src/components/VmDetails/cards/SnapshotsCard/SnapshotDetail.js

```diff
@@ -27,16 +27,16 @@ function getStatus (status) {
     : <span className={style['status-icon']}><Icon type='pf' name='off' />{msg.off()}</span>
 }
 
-const diskRender = (disk) => {
+const diskRender = (idPrefix, disk) => {
   const provSize = userFormatOfBytes(disk.get('provisionedSize'))
   const actSize = userFormatOfBytes(disk.get('actualSize'), provSize.suffix)
-  return <div key={disk.get('id')}>
+  return <div key={disk.get('id')} id={`${idPrefix}-${disk.get('name')}`}>
```

Comment:
What if 2 disks have the same name?

Reply:
Hmm... interesting, disks can have same names. I don't see why anyone can use this 'feature', but you are right.

Reply:
In general, if people can do it, they will. :wink: 

Using the name+index is ok but won't be stable.  If a new disk is added with the same name, they are sort equivalent in terms of name but wouldn't necessarily be sorted in the same order each time.  If that's ok for testing purposes, then we're good to go.

Reply:
I think for testing purposes they will use different names, thus everything alright.